src/constructors.ts

```diff
@@ -1,10 +1,10 @@
 import { ContextError, ErrorList, InputError } from './errors.ts'
 import type { Internal } from './internal/types.ts'
+import type { StandardSchemaV1 } from '@standard-schema/spec'
```

Comment:
What do you think of keeping the V1 only in the import? Looks a bit cleaner and reduces the delta in the case of upgrades.

```suggestion
import type { StandardSchemaV1 as StandardSchema } from '@standard-schema/spec'

```